chainermn/communicators/pure_nccl_communicator.py

```diff
@@ -157,6 +144,38 @@ def _unpack_params_from_buffer(self, params,
                 params, 'grad', self.gpu_buffer_a,
                 allreduce_grad_dtype, stream)
 
+    def multi_node_mean_nccl(self, gpu_buffer_a, gpu_buffer_b,
+                             n_elems, dtype, stream=None):
+        # Performs allreduce and division by size, i.e. mean.
+        # gpu_buffer_a = Sigma(gpu_buffer_a, all-procs) / self.size
+        # b is just used as buffer
+        if chainer.is_debug():
+            stream.synchronize()
+            array_a = gpu_buffer_a.array(n_elems, dtype=dtype)
+            array_b = gpu_buffer_b.array(n_elems, dtype=dtype)
+            self.check_ready_to_allreduce(array_a, array_b)
+
+        if stream is None:
+            stream = chainer.cuda.Stream.null
+        self._init_comms()
+        type_id = _communication_utility._get_nccl_type_id(dtype),
```

Comment:
`,`  at the end of the line is unnecessary

Reply:
Fixed!